app/swagger/requests/intent_to_file.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Requests
+    class IntentToFile
+      include Swagger::Blocks
+
+      swagger_path '/v0/intent_to_file' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Get a list of all Intent To File requests made by the veteran'
+          key :operationId, 'getIntentToFile'
+          key :tags, %w[benefits_forms]
```

Comment:
I'm not sold on this tag... but I dont know where else the ITF endpoints would belong. Open to suggestions.

Reply:
Perhaps a `form_526` would be appropriate.  There's a number of new endpoints being added for this form that can all be grouped.  What I'm not sure of, is how this would group with the existing disability claims swagger...